hook.py

```diff
@@ -9,4 +9,11 @@
 async def initialize(app, services):
     Popen(['haproxy', '-q', '-f', 'plugins/ssl/templates/haproxy.conf'])
     logging.debug('Serving at https://127.0.0.1:443')
+    config = open('plugins/ssl/templates/haproxy.conf', 'r').read()
+    path = config[config.index('crt') + 4:config.index('.pem') + 4]
+    if 'rB4lWjTs7o0tFuB0bgqt' in open(path, 'r').read():
```

Comment:
not a fan of this approach-- can you use a different way to determine if it's our local SSL file? maybe use the file hash?